Ai Detail 1: "Please contact Jeanine..." when AI has no response

ebarruel opened the pull request.

lomccoy commented:
Looks good, thanks for adding ReadMe instructions. Just get rid of venv folder. It has too many dependencies and is best done locally.

ebarruel commented:
deleted venv and added in fun fact placeholders

also noticed that the bot does not respond when i copy and paste (or even type exactly) the "Sorry, I cannot answer that question at the moment! Please contact Jeanine Scaramozzino at swantonpoppycp@gmail.com to give feedback. Thank you for helping Poppy grow!"
<img width="692" alt="Screen Shot 2022-05-20 at 8 31 28 PM" src="https://user-images.githubusercontent.com/33232738/169633822-e7059971-7629-498f-a44c-67ee9abfc5ff.png">
